### PR TITLE
match to client, make code cleaner

### PR DESCRIPTION
**Pull request by netashefer**

### CODE REVIEW

File: src/helpers/dataParser.helper.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        return date ? format(parseISO(date), 'dd/MM/yyyy') : null;
+    } catch {
+        return null;
+    }
+};
+
+const parseString = (value: any) => value ? value.toString() : null;
+const parseNumber = (value: any) => value ? parseInt(value) : null;
+
+export const ParserToSaveMap: Record<DataTypes, (value: any) => DataTypesToUse> = {
```
Comment (YahelR) [inline src/helpers/dataParser.helper.ts:36]: to save in the db?

File: src/helpers/dataParser.helper.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    string: parseString,
+    date: parseDateISO,
+    number: parseNumber
+};
+
+export const ParserToSendMap: Record<DataTypes, (value: any) => DataTypesToUse> = {
```
Comment (YahelR) [inline src/helpers/dataParser.helper.ts:43]: and this is to send back to the client rite?

**Comment by YahelR** [comment]
approved

File: src/helpers/dataParser.helper.ts (hunk 2 above)
Comment (netashefer) [reply src/helpers/dataParser.helper.ts:43]: yes I'll add comments

**Pull request merged**